src/fixate/drivers/dmm/fluke_8846a.py

```diff
@@ -5,38 +5,6 @@
 from fixate.drivers.dmm.helper import DMM
 import time
 
-MODES = {
-    ":VOLTage": {
-        ":DC": {"[:RATio]": {}, " [{range}]": {", [{resolution}]": {}}},
-        ":AC": {" [{range}]": {", [{resolution}]": {}}},
-    },
-    ":CURRent": {
-        ":DC": {" [{range}]": {", [{resolution}]": {}}},
-        ":AC": {" [{range}]": {", [{resolution}]": {}}},
-    },
-    ":RESistance": {" [{range}]": {", [{resolution}]": {}}},
-    ":FRESistance": {" [{range}]": {", [{resolution}]": {}}},
-    ":FREQuency": {" [{range}]": {", [{resolution}]": {}}},
-    ":PERiod": {" [{range}]": {", [{resolution}]": {}}},
-    ":CAPacitance": {" [{range}]": {", [{resolution}]": {}}},
-    ":TEMPerature": {":FRTD": {" [{RTD_type}]": {}}, ":RTD": {" [{RTD_type}]": {}}},
```

Comment:
Somewhere along the line we've lost the temperature measurement in this driver, may not be important as it was likely never used

Reply:
Yes. Temperature was neglected because it's not used.